routing_model/index.html

```diff
@@ -132,8 +141,58 @@ <h2>Node as a member of a section</h2>
+  </p>
   <h2>Processing a candidate trying to join our section</h2>
+  <p>This flow represents what we do when a candidate contacts us to join our section.<br>
+  The process starts as we receive an ExpectCandidate RPC from this node.<br>
+  We vote for it in PARSEC to be sure all members of the section process it in the same order.<br>
+  Once it reaches consensus, we are ready to process that candidate by letting them perform the resource_proof (see AcceptAsCandidate).<br>
+  There are some subtleties, such as the fact that we only want to process one candidate at a time, but this is the general idea.
+  </p>
+  <button class="collapsible">RPC: ExpectCandidate</button>
+  <div class="content">
+    <p>We receive this RPC from a node that wants to join our section.<br>
+    They aren't connected to us, yet; so the RPC reaches us through their proxy node.<br>
+    On receiving it, we vote for ParsecExpectCandidate to process it in the same order as other members of our section.<br>
+    It kickstarts the entire chain of events in this diagram.
+    </p>
+  </div>
+  <button class="collapsible">RPC: ResourceProofResponse/<br>CandidateInfo</button>
+  <div class="content">
+  <p>These RPCs should generally be handled during resource proof, in AcceptAsCandidate.<br>
+  It could be that they reach us before we are ready to receive them. For instance, because although enough other nodes got consensus on ParsecExpectCandidate, I'm still a bit behind.<br>
+  It's OK to discard the RPC in this case as the node who aims to be rellocated here will perceive this as a timeout from our end and resend later.
+  </p>
+  </div>
+  <button class="collapsible">ParsecOnline<br>ParsecPurgeCandidate</button>
+  <div class=content>
+    <p>These RPCs come from AcceptAsCandidate.<br>
+    The same node could be accepted by some nodes who would vote ParsecOnline, but also time out for some other nodes who would vote for ParsecPurgeCandidate.<br>
+    If it's the case, we only want to process the first of these two events and discard the other one.<br>
+    Because the "Wait for 2" loop in AcceptAsCandidate has higher precendence than the "Wait for 1" loop in this one, this is exactly what will happen.
+    </p>
+  </div>
+  <button class="collapsible">shorter_prefix_section</button>
+  <div class="content">
+    <p>If we know of a section that has a shorter prefix than ours, we prefer for them to receive this incoming node rather than ourselves as it will help kepp the Network's sections tree balanced.<br>
+    This shorter_prefix_section is a function that will return None if we are the shortest of any section we know, Some if there is a better candidate.<br>
+    If it is Some, we will relay the RPC to them instead of ourselves.
+    </p>
+  </div>
+  <button class="collapsible">PROCESSING_CANDIDATE</button>
+  <div class="content">
+    <p>We want to allow at most one node at a time to enter our section through rellocation.<br>
```

Comment:
We want to relocate at most one node at a time to our section.

Reply:
Uhm. I was trying to emphasize the "incoming" aspect... But I guess yours reads better...
Maybe some middle-ground:
> We want to accept only one incoming relocation at a time into our section.